phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -2285,6 +2286,7 @@ public MetaDataResponse call(MetaDataService instance) throws IOException {
                     builder.setClientVersion(VersionUtil.encodeVersion(PHOENIX_MAJOR_VERSION, PHOENIX_MINOR_VERSION, PHOENIX_PATCH_NUMBER));
                     if (parentTable!=null)
                         builder.setParentTable(PTableImpl.toProto(parentTable));
+                    builder.setAddingColumns(addingColumns);
```

Comment:
Unable to understand the need of this field addingColumns in builder ? If we need this, then why not a corresponding field in DropColumn request.

Reply:
In a drop column request, the answer is always "true" so there's no need to set the variable; it's hard-coded when the MetadataEndpointImpl creates the DropColumnMutator. (AddColumnRequests can either be to add columns or alter table properties, and we need to distinguish to know whether to update the DDL timestamp; a drop is always a drop.)  

I'll remove the (useless) check on the variable in DropColumnMutator to make that more clear.

Reply:
Thank you for educating me. :)